CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## master
 
+## [0.4] - 2020-12-22
+- [added] Multiple lexicons can be save and stored (source and destination languages are unique together)
+- [changed] Now there can not be a duplicated word in a lexicon. (unique together lexicon and term)
+- [changed] Avoid repeated entries. (unique together word, translation and variation) 
+- [changed] Importation require to be passed a lexicon_code of an already existing lexicon.
```

Comment:
Typo: Importation **requires** to be passed a lexicon_code of an already existing lexicon.

Reply:
Fixed, thank you for finding it